backend/prompt_studio/prompt_studio_index_manager/models.py

```diff
@@ -90,31 +91,40 @@ class Meta:
         ]
 
 
+def delete_from_vector_db(index_ids_history, vector_db_instance_id):
+    org_schema = connection.tenant.schema_name
+    util = PromptIdeBaseTool(log_level=LogLevel.INFO, org_id=org_schema)
+    vector_db = VectorDB(
+        tool=util,
+        adapter_instance_id=vector_db_instance_id,
+    )
+    for index_id in index_ids_history:
+        logger.debug(f"Deleting from VectorDB - index id: {index_id}")
+        try:
+            vector_db.delete(ref_doc_id=index_id)
```

Comment:
@athul-rs can we test this explicitly for pinecone?  Because that is the only adapter where we have our implementation of delete today. Just to make sure it is working... Anyway the same delete is also being called in re-indexing flow. So this is not the first use case. But still...

Reply:
@nehabagdia @gaya3-zipstack Having an issue in local setup to test pinecone, will try to test it out in staging after deployment.

Reply:
Will need to test in Staging for Pinecone.